apps/web/components/Organization/OrganizationIslandFooter/OrganizationIslandFooter.css.ts

```diff
@@ -0,0 +1,51 @@
+import { themeUtils } from '@island.is/island-ui/theme'
+import { style } from '@vanilla-extract/css'
+
+export const contentContainer = style({
+  display: 'flex',
+  ...themeUtils.responsiveStyle({
+    xs: {
+      flexFlow: 'column nowrap',
+    },
+    lg: {
+      flexFlow: 'row nowrap',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+  }),
+})
+
+export const globeIcon = style({
+  width: '15px',
+  height: '15px',
+})
+
+export const langageToggleContainer = style({
```

Comment:
Possible typo? langage -> language
```suggestion
export const languageToggleContainer = style({
```

Reply:
You are correct, I've fixed the typo